<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourierClass</class>
 <widget class="QWidget" name="CourierClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Courier</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zona:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>All streets</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="deliverPushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Deliver</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
